config-scripts/cups-opsys.m4

```diff
@@ -37,9 +37,10 @@ fi
 AC_SUBST(LOCALTARGET)
 
 AC_MSG_CHECKING(for codesign utility)
-CODE_SIGN="/usr/bin/true"
-AC_SUBST(CODE_SIGN)
 if test "$host_os_name" = darwin; then
 	CODE_SIGN="/usr/bin/codesign"
+else
```

Comment:
I think I'll use:

    AC_PATH_PROGS(CODE_SIGN, codesign true)

Reply:
what if none is found? I think we should explicitly fail the build